python-bindings/test.py

```diff
@@ -335,11 +336,24 @@ def test_aggregate_verify_zero_items():
     assert AugSchemeMPL.aggregate_verify([], [], G2Element())
 
 
+def test_pickle_support():
+    seed = bytes([
+        0, 50, 6, 244, 24, 199, 1, 25, 52, 88, 192, 19, 18, 12, 89, 6,
+        220, 18, 102, 58, 209, 82, 12, 62, 89, 110, 182, 9, 44, 20, 254, 22
+    ])
```

Comment:
With this change, This binary format is part of our state that would need to be migrated if it ever changes. Is pickle a stable format? If it can change between python versions it seems risky to depend on.

There should probably be a comment here saying that these bytes may not change. If they do we'll have a migration task as well.

Reply:
This is only used as parameter for `AugSchemeMPL.key_gen` to generate a `PrivateKey`. I don't understand your concerns about changing bytes here. Can you elaborate?

Reply:
Oh, I didn't read this very carefully :/

I was reading this as the pickled bytes and the test to ensure it was de-pickled correctly. Since the binary format for this needs to be stable, I think a test like that would be good. It would alert us of any changes that alters the file format.

Reply:
Makes sense, see https://github.com/Chia-Network/bls-signatures/compare/20d3768ea2b2f6df6f941c51ce55022bbd913566..4740720265802ae42bc47eac05b73e5cd17891df

Reply:
Hmm dropped it again because it appears to create different results on CI for whatever reason.. i spent too much time with this stuff already and i don't think its super important to have this test but if you have something which works locally and on CI feel free to provide a commit.

Reply:
You might consider a few more tests of round-trips, especially of the `infinity` value (since it serializes in a unique way).